components/NewIssue/NewIssue.tsx

```diff
@@ -5,7 +5,7 @@ const NewIssue:FC<{}> = () => {
   return (
     <p className="pt-6 md:p-0 coming-soon">
       <span className="text-2xl">Be the first to add by creating a GitHub issue</span>{" "}
-      <span className="animate-pulse text-2xl text-purple-500">
+      <span className="motion-safe:animate-pulse text-2xl text-purple-500">
```

Comment:
It's important to respect the user's `prefers-reduced-motion` so that the page doesn't show long-running animations if doing so would distract the user, overload their device, or otherwise harm their ability to use the webpage.

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion

The `motion-safe` Tailwind feature is a nice way to gate on `prefers-reduced-motion`: https://tailwindcss.com/docs/animation#prefers-reduced-motion

Reply:
I see, thanks